src/http/routing.rs

```diff
@@ -655,6 +655,13 @@ impl Route {
         format!(api!("/webhooks/{}/{}?wait={}"), webhook_id, token, wait)
     }
 
+    pub fn webhook_message<D>(webhook_id: u64, token: D, message_id: u64) -> String
```

Comment:
We should add documentation to this function.

Reply:
I didn't add documentation to the function since the rest of the routing functions themselves didn't have any documentation, and I didn't want to be inconsistent by documenting just one function. Would it be fine to make an exception just for this, or document all of the routing functions?

Reply:
Oh, you are right. Let's reconsider this later :)